test/cram/prefix_favouritism.t

```diff
@@ -105,10 +105,10 @@
   115 val Base.List.ignore_m : 'a t -> unit t
   116 val Base.List.drop : 'a t -> int -> 'a t
   116 val Base.List.take : 'a t -> int -> 'a t
-  125 mod Base.ListLabels
```

Comment:
Thats weird, why did this show up before ? Its not part of Base. Is the Stdlib included in Base ?

Reply:
Yes - [here](https://github.com/janestreet/base/blob/master/src/base.ml#L31-L77). The diff is because of a bugfix in the indexing code where we were not correctly ignoring this include, which should have been hidden due to the 'doc stop comment' [here](https://github.com/janestreet/base/blob/master/src/base.ml#L26)